draft-ietf-rtcweb-jsep.xml

```diff
@@ -3953,34 +3909,14 @@ Original:
 	       values are specified, include the RID header extension in
 	       the outgoing RTP streams, as indicated in
 	       <xref target="RFC8851" sectionFormat="comma" section="4"/>.</li>
-		 <li>If (1) the "m=" section is of type "audio" and (2)&nbsp;silence
-	       suppression was configured for the send media format as a
-	       result of processing the remote description and is also
-	       enabled for that format in the local description, use
+		 <li>If the "m=" section is of type "audio", and silence
+	       suppression was (1) configured for the send media format as a
+	       result of processing the remote description and (2) also
+	       enabled for that format in the local description, then use
```

Comment:
suggest dropping the 'then', as it's not used in the other bullets.

Reply:
'then' removed.